### ISSUE TITLE
`git-extras update` disregards aliases

### ISSUE THREAD

**Issue by apjanke** [tj/git-extras#456]
If I install git-extras manually, I get prompts for conflicting aliases, and can skip those.

```
~/local/opp/git/git-extras
[~/local/opp/git/git-extras on ⇄ relocatable-etc]
$ make PREFIX=/tmp/gitex SYSCONFPREFIX=/tmp/gitex install
... installing bins to /tmp/gitex/bin
... installing man pages to /tmp/gitex/share/man/man1
... installing git-archive-file
... installing git-authors
... installing git-back
git-bug conflicts with an alias, still install it and disable the alias? [y/n]n
... installing git-changelog
```

With `git-extras update`, it doesn't issue the prompt, and just clobbers all the aliases with commands.

```
$ git config --get-regexp 'alias.*'
alias.bug git checkout
[~]
$ git-extras update
...
... installing git-back
... installing git-bug
... installing git-changelog
... installing git-chore
... installing git-commits-since
```

I suspect it's because the `curl -s | bash` form that [`git-extras update` uses](https://github.com/tj/git-extras/blob/5fb7f6b77c485bb583666193e10f69e5d8e19739/bin/git-extras#L6-L14) from #381 is incompatible with the prompting the `Makefile` does: it can't read user response from `stdin` because the script it's executing is on `stdin` instead.

**Comment by hemanth** [comment]
//cc @spacewander

**Comment by apjanke** [comment]
The alias detection seems a bit tricky in other ways. What gets installed depends on what user you install it as, since git configuration is per-user. And if you're in a repo when you do the installation, it will detect aliases local to that repo, too. (Could be fixed by doing `--global` with the `config --get-regexp`.) I wonder if there's a runtime way to prioritize user aliases instead?

**Comment by nicolaiskogheim** [comment]
> I wonder if there's a runtime way to prioritize user aliases instead?

No, [a git command will shadow an alias](I wonder if there's a runtime way to prioritize user aliases instead?).

**Comment by spacewander** [comment]
A runtime way to prioritize user aliases...
Err, we can put something at the head of scripts like `is_git_repo`, and detect if an user alias of the same name exists. But I don't think it is a good idea.

I just got a new idea about alias conflict avoidance. Instead of prompting it in the Makefile, we can just throw out warning, and tell the users how to remove conflicted files.

**Comment by raviranjananalist** [comment]
For Expert 5 Packers and Movers Hyderabad go to  
http://www.expert5th.in/packers-and-movers-hyderabad/
For Expert 5 Packers and Movers Bangalore go to 
http://www.expert5th.in/packers-and-movers-bangalore/
For Expert 5 Packers and Movers Mumbai go to 
http://www.expert5th.in/packers-and-movers-mumbai/

**Comment by raviranjananalist** [comment]
For Expert 5 Packers and Movers Gurgaon go to 
http://www.expert5th.in/packers-and-movers-gurgaon/
For Expert 5 Packers and Movers Delhi go to 
http://www.expert5th.in/packers-and-movers-delhi/
For Expert 5 Packers and Movers Pune go to  
http://www.expert5th.in/packers-and-movers-pune/

**Comment by raviranjananalist** [comment]
For Expert 5 Packers and Movers Chennai go to 
http://www.expert5th.in/packers-and-movers-chennai/
For Expert 5 Packers and Movers Noida go to 
http://www.expert5th.in/packers-and-movers-noida/
For Expert 5 Packers and Movers Thane go to 
http://www.expert5th.in/packers-and-movers-thane/
For Expert 5 Packers and Movers Navi Mumbai go to 
http://www.expert5th.in/packers-and-movers-navimumbai/
For Expert 5 Packers and Movers Ghaziabad go to 
http://www.expert5th.in/packers-and-movers-ghaziabad/
For Expert 5 Packers and Movers Faridabad go to 
http://www.expert5th.in/packers-and-movers-faridabad/

**Comment by raviranjananalist** [comment]
For Expert 5 Packers and Movers Bangalore go to 
http://www.expert5th.in/packers-and-movers-bangalore/
For Expert 5 Packers and Movers Pune go to  
http://www.expert5th.in/packers-and-movers-pune/
For Expert 5 Packers and Movers Mumbai go to 
http://www.expert5th.in/packers-and-movers-mumbai/
